python/pyspark/pandas/supported_api_gen.py

```diff
@@ -0,0 +1,356 @@
+                - set(signature(ps_func).parameters)
+                - COMMON_PARAMETER_SET
+            )
+            if missing_set:
+                # partially implemented
+                pd_dict[pd_func_name] = SupportedStatus(
+                    Implemented.PARTIALLY_IMPLEMENTED.value,
+                    transform_missing(
+                        module_name,
+                        pd_func_name,
+                        missing_set,
+                        pd_module_group.__name__,
+                        ps_module_group.__name__,
+                    ),
+                )
+            else:
+                # implemented including it's whole parameter
+                pd_dict[pd_func_name] = SupportedStatus(Implemented.IMPLEMENTED.value)
+        else:
+            # not implemented yet
+            pd_dict[pd_func_name] = SupportedStatus(Implemented.NOT_IMPLEMENTED.value)
+    return pd_dict
+
+
+def transform_missing(
+    module_name: str,
+    pd_func_name: str,
+    missing_set: Set[str],
+    pd_module_path: str,
+    ps_module_path: str,
+) -> str:
+    """
+    Transform missing parameters into table information string.
+
+    Parameters
+    ----------
+    module_name : str
+        Class name that exists in the path of the module.
+    pd_func_name : str
+        Name of pandas API.
+    missing_set : Set[str]
+        A set of parameters not yet implemented.
+    pd_module_path : str
+        Path string of pandas module.
+    ps_module_path : str
+        Path string of pyspark.pandas module.
+
+    Examples
+    --------
+    >>> transform_missing("DataFrame", "add", {"axis", "fill_value", "level"},
+    ...                     "pandas.DataFrame", "pyspark.pandas.DataFrame")
+    '``axis`` , ``fill_value`` , ``level``'
+    """
+    missing_str = " , ".join(f"``{x}``" for x in sorted(missing_set)[:MAX_MISSING_PARAMS_SIZE])
+    if len(missing_set) > MAX_MISSING_PARAMS_SIZE:
+        module_dot_func = f"{module_name}.{pd_func_name}" if module_name else pd_func_name
+        additional_str = (
+            " and more. See the "
+            f"`{pd_module_path}.{module_dot_func} "
+            "<https://pandas.pydata.org/docs/reference/api/"
+            f"{pd_module_path}.{module_dot_func}.html>`__ and "
+            f"`{ps_module_path}.{module_dot_func} "
+            "<https://spark.apache.org/docs/latest/api/python/reference/pyspark.pandas/api/"
+            f"{ps_module_path}.{module_dot_func}.html>`__ for detail."
+        )
+        missing_str += additional_str
+    return missing_str
+
+
+def get_pd_modules(pd_module_group: Any) -> List[str]:
+    """
+    Returns sorted pandas memeber list from pandas module path.
+
+    Parameters
+    ----------
+    pd_module_group : Any
+        Specific path of importable pandas module.
+    """
+    return sorted([m[0] for m in getmembers(pd_module_group, isclass) if not m[0].startswith("_")])
+
+
+def update_all_supported_status(
```

Comment:
Just some think aloud here: Do we want to add a public interface to help dev to see which method/parameter are missing with pretty print, like: `get_supported_status("Dataframe", pd, ps)`?

If no need, just ignore this comment.
 
cc @HyukjinKwon

Reply:
I think the documentation here alone should be good enough. Other uses can use the methods here (but these are not API for clarification since it's not documented).